File: frontend/DictionaryApp/DictionaryApp/Models/PhraseResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DictionaryApp.Models
{
    public class PhraseResponse
    {
        public List<Phrase> directPhrases { get; set; }
        public List<Phrase> relatedPhrases { get; set; }
        public Word word { get; set; }
    }
}

File: backend/app/src/main/java/com/dictionary/app/Services/PhraseResponse.java
package com.dictionary.app.Services;

import com.dictionary.app.Models.Phrase;
import com.dictionary.app.Models.Word;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PhraseResponse {

    private List<Phrase> directPhrases;   // Directly related phrases
    private List<Phrase> relatedPhrases;  // Related phrases where word appears in content
    private Word word;                    // The word object associated with these phrases
}
